### PR TITLE
[Python3.11] Add support for Python 3.11 in SLES15 builds

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- Support python3.11 in sles15 build

## How was this patch tested?

- Locally
- sles15 docker image

### CODE REVIEW

File: tools/cloudera/build_hue_cloudera.sh [hunk 1]
```diff
@@ -51,8 +51,10 @@ function install_prerequisite() {
   export SQLITE3_PATH=${SQLITE3_PATH:-"$TOOLS_HOME/sqlite/sqlite3"}
   if [[ $1 == "redhat9" || $1 == "redhat9_ppc" ]]; then
     check_python39_path
-  elif [[ $1 == "ubuntu22" || $1 == "sles15" ]]; then
+  elif [[ $1 == "ubuntu22" ]]; then
     check_python310_path
+  elif [[ $1 == "sles15" ]]; then
```
Comment (athithyaaselvam) [inline tools/cloudera/build_hue_cloudera.sh:56]: So we don't support sles15 with python3.10 anymore? and it will only work with python3.11?

**Review by Harshg999 — COMMENTED** [review]
Left a review comment..

File: tools/cloudera/build_hue_cloudera.sh [hunk 2]
```diff
@@ -111,11 +113,16 @@ if [[ $DOCKEROS == "redhat9" || $DOCKEROS == "redhat9_ppc" ]]; then
...
   export SYS_PYTHON=$PYTHON39_PATH/bin/python3.9
   export PATH=$PYTHON39_PATH/bin:$PATH
-elif [[ $DOCKEROS == "ubuntu22" || $DOCKEROS == "sles15" ]]; then
+elif [[ $DOCKEROS == "ubuntu22" ]]; then
   export PYTHON_H=$PYTHON310_PATH/include/python3.10/Python.h
   export PYTHON_VER=python3.10
   export SYS_PYTHON=$PYTHON310_PATH/bin/python3.10
   export PATH=$PYTHON310_PATH/bin:$PATH
+elif [[ $DOCKEROS == "sles15" ]]; then
+  export PYTHON_H=$PYTHON311_PATH/include/python3.10/Python.h
```
Comment (Harshg999) [inline tools/cloudera/build_hue_cloudera.sh:122]: Path seems wrong, can you check it again?

File: tools/cloudera/build_hue_cloudera.sh (hunk 1 above)
Comment (agl29) [reply tools/cloudera/build_hue_cloudera.sh:56]: @athithyaaselvam Yes, we will now support py3.11 on sles15. All sles15 builds (internal) will be tied to py3.11.

**Review by Harshg999 — APPROVED** [review]
LGTM!

**Review by athithyaaselvam — APPROVED** [review]

**Pull request closed without merging**